kafka-ui-api/src/main/java/com/provectus/kafka/ui/service/MessagesService.java

```diff
@@ -48,6 +48,7 @@
 @Service
 @RequiredArgsConstructor
 @Slf4j
+@SuppressWarnings("squid:S2589")
```

Comment:
?

Reply:
The problem is the generated CreateTopicMessageDTO, which can actually return null.  We have tests that check this